nvi

```diff
@@ -100,24 +101,57 @@ function install_node() {
   mkdir --parents -- "$BIN_DIR"
 
   # Download and extract the tar if it doesn't exist on disk.
-  if [[ ! -d "$INSTALL_DIR/node-v$NODE_VERSION-linux-x64" ]]; then
+  if [[ ! -d "$INSTALL_DIR/$PACKAGE" ]]; then
     # Fail if version doesn't exist.
-    NODE_DIST_LOOKUP=$(wget -qO - https://nodejs.org/dist/index.json | grep --null-data --only-matching --perl-regex --text "\"version\"\s*:\s*\"v$NODE_VERSION\"")
-    [[ -z "$NODE_DIST_LOOKUP" ]] && >&2 echo "Error: Couldn't find Node.js version v$NODE_VERSION. Aborting." && exit 1
+    local NODE_DIST_LOOKUP
```

Comment:
I see. I think I prefer the last one, where each (flag, argument) pair is on the same line. It's a weak preference, though, and I'm fine with all of them.